### PR TITLE
feat: add more packed items

### PR DESCRIPTION
**Pull request by pyphilia**
Since I'm improving the frontend, I've noticed I needed more packed items from other endpoints:
- recycled
- bookmark
- liked
- collections (from member) 

Most probably we will want more of them.

### CODE REVIEW

**Review by ReidyT — APPROVED** [review]
Nice PR ! LGTM 😆

**Review by spaenleh — COMMENTED** [review]
Looks nice ! Just some comments, I let you check the tests, maybe rebase on the websockets test fixes.

File: src/services/item/plugins/itemFavorite/schemas.ts [hunk 1]
```diff
@@ -14,6 +14,19 @@ export default {
...
+    packedFavorite: {
+      type: 'object',
+      properties: {
+        id: {
+          $ref: 'https://graasp.org/#/definitions/uuid',
+        },
+        item: {
+          $ref: 'https://graasp.org/items/#/definitions/packedItem',
+        },
+        createdAt: { type: 'string' },
```
[the remark this answers is not in the document]
Comment (pyphilia) [reply src/services/item/plugins/itemFavorite/schemas.ts:26]: kinda, but you don't really need it, it's always your own favorites.

**Comment by pyphilia** [comment]
I've actually got a bit broader and added hidden and public tag in the packed item.

**Pull request merged**